src/sidebars/favourites/minichart/minichart-controller.js

```diff
@@ -16,6 +17,12 @@
                         var width = extent.width,
                             height = extent.height;
                         var data = result.data;
+
+                        if (data.length < 2) {
```

Comment:
There needs to be 2 data points to draw the line (otherwise it just does what is in the issue).